github-api-call-inventory/github-api-call-inventory.ipynb

```diff
@@ -0,0 +1,520 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b0f85bb2",
+   "metadata": {},
+   "source": [
+    "## Events"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "283cb1e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dataclasses import dataclass, fields\n",
+    "\n",
+    "@dataclass\n",
+    "class Event:\n",
+    "    is_project_private: int\n",
+    "    load_packit_yaml: int\n",
+    "    set_status: int\n",
```

Comment:
Is the GitHub API called for any other reason while handling PR events?

Reply:
iirc we also check whether the one who triggered the event has write access to repo or is author of the PR

Reply:
Where is this happening? I've looked in the packit-service code, but I was able to find such check only for vm_image jobs. What am I missing? :thinking:

Reply:
Found [it](https://github.com/packit/packit-service/blob/b946857a437b062d6e93021b5420bf46f3ce9964/packit_service/worker/jobs.py#L385).

Reply:
Added.